scripts/plugins/install_plugins.py

```diff
@@ -50,52 +104,26 @@ def install_plugins():  # pylint: disable=too-many-locals,too-many-statements
             sys.exit(1)
 
         # ~#~#~#~#~#~#~#~#~#~#~#~#~#
-        # CLONE PLUGIN FROM SOURCE
+        # CLONE PLUGIN FROM LOCAL SOURCE
         # ~#~#~#~#~#~#~#~#~#~#~#~#~#
-
-        plugin_tag = (
-            bitops_plugins_configuration[plugin_config].source_tag
-            if bitops_plugins_configuration[plugin_config].source_tag is not None
-            else "latest"
-        )
-        plugin_branch = (
-            bitops_plugins_configuration[plugin_config].source_branch
-            if bitops_plugins_configuration[plugin_config].source_branch is not None
-            else "main"
-        )
-
-        logger.info(
-            f"\n\n\n~#~#~#~CLONING PLUGIN [{plugin_config.upper()}]~#~#~#~  \
-        \n\t PLUGIN_SOURCE:         [{plugin_source}]               \
-        \n\t PLUGIN_TAG:            [{plugin_tag}]                  \
-        \n\t PLUGIN_BRANCH:         [{plugin_branch}]                \
-        \n#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~# \n"
-        )
-
-        try:
-            # Non-Entry default
-            if plugin_branch == "latest" and plugin_tag == "main":
-                git.Repo.clone_from(plugin_source, plugin_dir + plugin_config)
-
-            # If the plugin branch and tag are specified, default to branch
-            elif plugin_branch is not None and plugin_tag is not None:
-                git.Repo.clone_from(plugin_source, plugin_dir + plugin_config, branch=plugin_branch)
-
-            else:
-                plugin_pull_branch = plugin_tag if plugin_branch is None else plugin_branch
-                git.Repo.clone_from(
-                    plugin_source,
-                    plugin_dir + plugin_config,
-                    branch=plugin_pull_branch,
-                )
-
-            logger.info(f"\n~#~#~#~CLONING PLUGIN [{plugin_config}] SUCCESSFULLY COMPLETED~#~#~#~")
-
-        except Exception as err:
-            logger.error(
-                f"\n~#~#~#~CLONING PLUGIN [{plugin_config}] CRITICAL ERROR~#~#~#~\n\t{err}"
+        if plugin_source.startswith("file:///"):
```

Comment:
I think it's 
```suggestion
        if plugin_source.startswith("file://"):
```
as `//` is the prefix really.